cmd/flowserver/main.go

```diff
@@ -157,7 +95,7 @@ func jsonHandler(handler jsonHandlerFunc) http.HandlerFunc {
 		r = r.WithContext(context.WithValue(r.Context(), contextStart, time.Now()))
 		value, err := handler(w, r)
 		if err != nil {
-			writeError(w, r, http.StatusInternalServerError, err)
+			writeError(w, r, http.StatusBadRequest, err)
```

Comment:
Now any error is returned as a 400. `utils.ValidationErrors` is only really used to combine multiple field errors into a single error, and probably should never be returned directly since it always lack context, i.e. what object are those failures on.